system-tests/_support/utils/ServicesUtil.js

```diff
@@ -53,16 +50,23 @@ Cypress.Commands.add("configurationMapValue", { prevSubject: true }, function(
   const compareLabel = label.toLowerCase();
 
   elements.each(function(index, element) {
-    const labelElement = element.querySelector(".configuration-map-label");
-    if (!labelElement) {
-      return;
+    const rows = [...element.querySelectorAll("div > .configuration-map-row")];
```

Comment:
`div > .configuration-map-row` should be `SOME_CLASS > .configuration-map-row` to make more stable/future proof

Reply:
You mean like `* > .configuration-map-row` or some specific class?